ldap/Dockerfile

```diff
@@ -40,7 +40,7 @@ VOLUME ["/etc/ldap", "/var/lib/ldap"]
 
 ENTRYPOINT [ "/docker-entrypoint.sh" ]
 
-CMD [ "sh", "-c", "exec slapd -d 32768 -u ${RUN_AS_UID} -g ${RUN_AS_GID}" ]
+CMD [ "sh", "-c", "exec slapd -d ${SLAPD_LOG_LEVEL:-32768} -u ${RUN_AS_UID} -g ${RUN_AS_GID}" ]
```

Comment:
IIRC this new env var has to be defined in an ENV statement before (but docker is nice to us and does not bark)

Reply:
No, simply, if the SLAPD_LOG_LEVEL variable has been set, it's used, otherwise the default value (32768) is used as before